cmd/transportRequestUploadSOLMAN.go

```diff
@@ -82,52 +70,23 @@ func runTransportRequestUploadSOLMAN(config *transportRequestUploadSOLMANOptions
 		Password: config.Password,
 	})
 
-	cdID, err := getChangeDocumentID(config, &transportRequestUtils{})
```

Comment:
What would be wrong with keeping this here? I can imagine cases where the git repo is present at that time. In this case it would not be strongly required to call the scan-steps prior to this step and a - at this point still missing - ID can be retrieved here silently. With that there is no strong dependency to calling the scan-steps before. All we have to ensure that we fail with a reasonable error message in case neither the ID are on CPE nor any git repo is availabe.

Reply:
Separation of concerns: transportRequestUploadSOLMAN does nothing more than the upload. 
Means: clear usage, no stashing neccessary.
Actually, i felt relieved while splitting the source. so I'd keep it as it is.